front/lib/actions/helpers.ts

```diff
@@ -65,7 +58,8 @@ export async function callAction<V extends t.Mixed>({
     logger
   );
 
-  const r = await prodAPI.runApp(user, app, config, [input]);
+  // The doc tracker apps are running without any group privileges for now.
+  const r = await prodAPI.runApp([], app, config, [input]);
```

Comment:
Shouldn't this include at least the global group?

Reply:
Seen with @lasryaric in person, we will refactor the doc tracker to use auto everywhere instead. This was not part of this PR and will be a follow up PR.

Reply:
auto ?

Reply:
~auto~ auth